tests/test_cleanup.py

```diff
@@ -17,16 +16,20 @@
 _DAMAGED_ZIP_CONTENT = b"*BAD*file"
 
 
-def _zip_bytes() -> bytes:
+def wrapzip(filename: str, content: bytes) -> bytes:
+    """Create an in-memory zip archive with a single file"""
     bio = io.BytesIO()
     z = zipfile.ZipFile(bio, mode="w", compression=zipfile.ZIP_STORED)
-    z.writestr("content.txt", _ZIP_CONTENT)
+    z.writestr(filename, content)
     z.close()
     return bio.getvalue()
 
 
-ZIP_BYTES = _zip_bytes()
+ZIP_BYTES = b"prefix to force carving the zip archive " + wrapzip(
+    "content.txt", _ZIP_CONTENT
+)
 DAMAGED_ZIP_BYTES = ZIP_BYTES.replace(_ZIP_CONTENT, _DAMAGED_ZIP_CONTENT)
+assert ZIP_BYTES != DAMAGED_ZIP_BYTES
```

Comment:
This looks like an independent change from the topic of this commit. It is no big deal to have a few extra changes like that in a PR but I'd like you to at least put it into separate git commits. This would make reviewing changes much easier.